juntagrico/templates/admin/mark_share_intermediate.html

```diff
@@ -0,0 +1,32 @@
+{% extends "admin/base_site.html" %}
+{% load i18n %}
+{% load config %}
+
+{% block content %}
+    {% vocabulary "share" as v_share %}
+    {% vocabulary "share_pl" as v_share_pl %}
+<h1>{% blocktrans %}{{ v_share_pl }} bearbeiten{% endblocktrans %}</h1>
+<p>{% blocktrans %}Bist du sicher, dass du die ausgewählten {{ v_share_pl }} bearbeiten willst?{% endblocktrans %}</p>
+
+<h2>{% trans "Optionen" %}</h2>
+<form action="" method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+
+    <h2>{% trans "Zusammenfassung" %}</h2>
+    <ul>
+        <li>{{ v_share_pl }}: {{ shares|length }}</li>
+    </ul>
+
+    <h2>{% trans "Objekte" %}</h2>
+    <ul>
+        {% for share in shares %}
+        <li>{{ v_share }}: <a href="{% url 'admin:juntagrico_share_change' share.id %}">{{ share }}</a></li>
+        <input type="hidden" name="_selected_action" value="{{ share.pk }}" />
```

Comment:
wont that not get overwritten?

Reply:
This is how to transfer the pks through the action steps. Works as expected.